src/ontology/mondo-edit.obo

```diff
@@ -586007,6 +586015,14 @@ intersection_of: MONDO:0037939 ! porphyria
 intersection_of: has_characteristic MONDO:0021141 ! acquired
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-3458-4839
 
+[Term]
+id: MONDO:1040001
+name: TP63-related ectodermal dysplasia spectrum with limb and orofacial malformations
+def: "Any developmental defect during embryogenesis in which the cause of the disease is a mutation in the TP63 gene. Characterized by variable ectodermal dysplasia, limb defects, and orofacial clefting." [https://orcid.org/0000-0002-0587-4693, PMID:10535733, PMID:10839977, PMID:11159940, PMID:11462173, PMID:11528512, PMID:12766194, PMID:16740912, PMID:21204238, PMID:22574117]
```

Comment:
@yshwetar based on the PMIDs, can you confirm that "any development defect caused by a mutation in TP63" will always lead to this disease?  I am assuming it could be the case, but I want to make sure.